include/bluetooth/services/fast_pair.h

```diff
@@ -88,6 +157,22 @@ void bt_fast_pair_set_pairing_mode(bool pairing_mode);
  */
 bool bt_fast_pair_has_account_key(void);
 
+/** Set or update specified battery value.
+ *
+ * Battery values may be used to generate advertising packet. To include battery values in
+ * advertising packet this function must be called before @ref bt_fast_pair_adv_data_fill. It is
+ * user responsibility to update battery value and regenerate advertising packet when battery value
+ * change.
+ *
+ * @param[in] battery_value	Battery value.
+ * @param[in] battery_value_idx	Index of battery value.
+ *
+ * @retval 0 on success.
+ * @retval -EINVAL if battery value is invalid.
+ */
+int bt_fast_pair_set_battery_value(struct bt_fast_pair_battery_value battery_value,
+				   enum bt_fast_pair_battery_value_idx battery_value_idx);
```

Comment:
Please change the order of parameters. The object ID parameter should always be the first (think Bluetooth connection API in Zephyr):

```suggestion
int bt_fast_pair_battery_value_set(enum bt_fast_pair_battery_value_idx battery_value_idx,
				   struct bt_fast_pair_battery_value battery_value);
```

Reply:
Ok, I will chenage the order of parameters, but I have concerns regarding this conception of verb at the end of function name. If we want to do it we should create separate PR and align it everywhere in Fast Pair. Otherwise, we won't be consistent. For example in this API we have `bt_fast_pair_set_pairing_mode` so it's natural to have `bt_fast_pair_set_battery_value`.

Reply:
You are right, let's keep it as it is.